features/overrides.feature

```diff
@@ -0,0 +1,101 @@
+Feature: Implementing Runtime Overrides for Select Configuration Keys
+
+        Scenario: Override configured editor with built-in input === editor:''
+        Given we use the config "basic_encrypted.yaml"
+        And we use the password "test" if prompted
+        When we run "jrnl --config-override editor ''"
+        Then the stdin prompt should have been called
+        
+        Scenario: Postconfig commands with overrides
+        Given We use the config "basic_encrypted.yaml"
+        And we use the password "test" if prompted
+        When we run "jrnl --decrypt --config-override highlight false --config-override editor nano"
+        Then the config should have "highlight" set to "bool:false"
+        And no editor should have been called
+        
+        @skip_win
+        Scenario: Override configured linewrap with a value of 23
+        Given we use the config "simple.yaml"
+        And we use the password "test" if prompted
+        When we run "jrnl  -2 --config-override linewrap 23 --format fancy"
+        Then the output should be
+
+        """
+        ┎─────╮2013-06-09 15:39
+        ┃ My  ╘═══════════════╕
+        ┃ fir st  ent ry.     │
```

Comment:
The `fancy` format looks like it has some issues. We need to fix this (not in this PR; just putting a note so we don't forget).

Reply:
created #1207 so there's something to mark this resolved